crates/biome_lsp/src/session.rs

```diff
@@ -311,6 +311,7 @@ impl Session {
                 path: biome_path.clone(),
                 categories,
                 max_diagnostics: u64::MAX,
+                rule: None,
```

Comment:
Here I set `rule` to `None` because I do not have access to this data. I am not sure about the consequences. Maybe it only affects the LSP. Since the CLI is the only one using the `rule` option, it's probably OK.